gsa/src/web/pages/tasks/listpage.js

```diff
@@ -162,6 +158,23 @@ const TasksListPage = () => {
     filter,
     changeFilter,
   );
+  const [tagsDialogVisible, setTagsDialogVisible] = useState(false);
+
+  // Task list state variables and methods
+  const [
+    getTasks,
+    {counts, tasks, error, loading: isLoading, refetch, called, pageInfo},
+  ] = useLazyGetTasks();
+
+  const exportTasksByFilter = useExportTasksByFilter();
+  const exportTasksByIds = useExportTasksByIds();
+  const bulkExportTasks = useBulkExportEntities();
+
+  const [deleteTask] = useDeleteTask();
+const [deleteTasksByIds] = useDeleteTasksByIds();
+const [deleteTasksByFilter] = useDeleteTasksByFilter();
```

Comment:
Not sure why prettier didn't fix the indentation here.
```suggestion
    const [deleteTasksByIds] = useDeleteTasksByIds();
    const [deleteTasksByFilter] = useDeleteTasksByFilter();
```